### PR TITLE
feat: track anonymous personfull events in usage reports

### PR DESCRIPTION
**Pull request by raquelmsmith**
## Problem

We want to see how many of our events per customer are anonymous but personfull to see if we can make the anonymous events all personless.

## Changes

In usage reports, adds a metric for anonymous personfull events. 

I will re-run usage reports for the last few days so this metric shows up and we can run numbers from there. 

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## How did you test this code?

Updated the tests, which annoyingly took forever, I think there are too many ways to do the same thing, or they're named poorly or something.

### CODE REVIEW

File: posthog/tasks/usage_report.py [hunk 1]
```diff
@@ -416,6 +417,40 @@ def get_teams_with_billable_enhanced_persons_event_count_in_period(
...
+        distinct_expression = "1"
+
+    result = sync_execute(
+        f"""
+        SELECT team_id, count({distinct_expression}) as count
+        FROM events
+        WHERE timestamp between %(begin)s AND %(end)s
+            AND event != '$feature_flag_called' AND event NOT IN ('survey sent', 'survey shown', 'survey dismissed')
+            AND person_mode IN ('full', 'force_upgrade')
+            AND JSONExtractBool(properties, '$is_identified') = 0
```
Comment (raquelmsmith) [inline posthog/tasks/usage_report.py:444]: @robbie-c what if someone had used `$set` to add person properties - would they be considered identified from posthog.js?

File: posthog/models/event/util.py [hunk 2]
```diff
@@ -224,11 +224,12 @@ def bulk_create_events(
 
                 except Group.DoesNotExist:
                     continue
+        properties = event.get("properties", {})
```
Comment (zlwaterfield) [inline posthog/models/event/util.py:227]: This doesn't seems like it didn't anything.

File: posthog/tasks/usage_report.py [hunk 3]
```diff
@@ -74,6 +74,7 @@ class UsageReportCounters:
     event_count_in_period: int
     enhanced_persons_event_count_in_period: int
     event_count_with_groups_in_period: int
+    anonymous_personfull_event_count_in_period: int
```
Comment (zlwaterfield) [inline posthog/tasks/usage_report.py:77]: Nit: shouldn't this have one l? "personful"

**Review by zlwaterfield — APPROVED** [review]
Just added two questions

File: posthog/tasks/usage_report.py (hunk 3 above)
Comment (raquelmsmith) [reply posthog/tasks/usage_report.py:77]: We've used "personfull" in internal communications but I think it's a britishism, so yes changing to `personful` 😄

**Pull request merged**